watson-developer-cloud/discovery-components
# fix: make DocPrev only display passage text with highlighting

#### What do these changes do/fix?

https://github.ibm.com/Watson-Discovery/docviz-squad-issue-tracker/issues/219

This temporarily removes views from the DocumentPreview component that are not going to be used for now.  Also temporarily removes storybook entries and tests related to those views.

#### How do you test/verify these changes?

Run storybook and go to `DocumentPreview`.  See that only the `SimpleDocument` view is present along with passage highlighting in the view.
